nflow-engine/src/main/java/io/nflow/engine/internal/dao/WorkflowInstanceDao.java

```diff
@@ -801,7 +838,8 @@ public String getWorkflowInstanceState(long workflowInstanceId) {
           .setModified(sqlVariants.getDateTime(rs, "modified")) //
           .setStartedIfNotSet(sqlVariants.getDateTime(rs, "started")) //
           .setExecutorGroup(rs.getString("executor_group")) //
-          .setSignal(ofNullable(getInt(rs, "workflow_signal")));
+          .setSignal(ofNullable(getInt(rs, "workflow_signal"))) //
+          .setArchived(rs.getBoolean("archived"));
```

Comment:
Do we have `archived` field in workflow table?

Reply:
No. The code adds it there as a virtual hard coded field in the select SQL, for example `select *, 1 as archived`